test/utils/test_merge_concat.py

```diff
@@ -0,0 +1,11 @@
+from scripts import utils
+
+
+def test_merge_concat():
```

Comment:
you should try to use the script with a value of different type (eg: integer or date)

Reply:
done!